app/views/content/life-as-a-teacher/explore-subjects/religiouseducation.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think the URL should be changed to `/explore-subjects/religious-education` - we usually separate distinct words